src/Arbitrum.js

```diff
@@ -166,9 +173,9 @@ function Arbitrum() {
 
   return (
     <div className="Home">
-      <h1>GMX Analytics / Arbitrum</h1>
+      <h1>Analytics / Arbitrum</h1>
       {lastSubgraphBlock && lastBlock &&
-        <p className={isObsolete ? 'warning' : ''} style={{ marginTop: '-1rem' }}>
+        <p className={cx('page-description', (isObsolete ? 'warning' : ''))} style={{ marginTop: '-1rem' }}>
```

Comment:
with `classnames` it's possible to write `cx('page-description', {warning: isObsolete})`, maybe a bit more convenient
fyi

Reply:
Fixed.